sql/sql_yacc.yy

```diff
@@ -9054,6 +9054,17 @@ query_expression:
           {
             $$= NEW_PTN PT_query_expression($1, $2, $3, $4);
           }
+        | query_expression_body
+          opt_order_clause
+          opt_limit_clause
+          into_clause
+          opt_locking_clause_list
+          {
+            if(!$1->set_into($4)) {
```

Comment:
We check here for `!xxx->set_into(...)` (with negation) to return parse error whereas comment for `set_into()` says `true if couldn't set`.

Reply:
Fixed, the comment was wrong.